content/docs/concepts/oracle-network/_index.md

```diff
@@ -0,0 +1,161 @@
+
+Autonity provides consensus-computed median price data for selected currency pairs as an L1 platform feature. The submission of price data for aggregation and voting is a validator responsibility and executed according to an _oracle protocol_. As consequence, validator nodes form an _oracle network_. When serving as a member of the [consensus committee](/glossary/#consensus-committee), validators must submit raw price data reports for currency pairs on-chain, and participate in the oracle protocol to agree an aggregated median price for the submitted price data in discrete _voting rounds_ using a _commit and reveal_ protocol. As voting rounds complete, the aggregated median price data computed and provided by the Autonity network is updated. As [epoch](/glossary/#epoch)'s end the set of oracle voters changes to accord with new consensus committee membership.
+
+In the oracle protocol a validator has responsibilities to:
+
+- Source raw price data for selected currency pairs from one-or-more external data sources off-chain.
+- Provide price data reports as input for the computation of aggregated median price data for currency pairs during oracle voting rounds.
+- Vote for aggregated median price data for currency pairs during oracle [voting rounds](/glossary/#voting-round).
+- Maintain the set of symbols for which raw price data is provided by the validator oracle, revising as required if the connected Autonity network updates its selected set of currency pairs.
+
+
+### Oracle prerequisites   
+
+To operate as a validator node the operator must operate Autonity [oracle server](/concepts/oracle-server/) software as an adjunct to its Autonity [full node](/concepts/client/) software.
+
+Prerequisites for participating in the oracle network as a validator node operator are:
+
+- An [Autonity Go Client](/concepts/client/) full node installation, joined to an Autonity network.
+- An [Autonity Oracle Server](/concepts/oracle-server/) installation, configured to connect to your full node and external price data sources via plugin adaptors.
+- Plugin adaptors for connecting to external data provider sources to retrieve price data for the currency pairs required for your node's connected Autonity network.
+
+## Oracle identity, accounts and keypairs
+The oracle server makes use of a single account and private/public [key pair](/glossary/#key-pair) for oracle identity, oracle protocol participation, and cryptographic security. 
+
+### Oracle server key
+
+The private/public key pair of the oracle server node.
+
+The private key is used:
+
+- By an oracle server for submitting price report transactions to the Oracle Contract on-chain.
+- To generate the `proof` of enode ownership required for validator registration. The `proof` is generated using the [`genOwnershipProof`](/reference/cli/agc/#command-line-options) command-line option of the Autonity Go Client. 
+
```

Comment:
It confuse me here, in my understanding, the enode proof are generated by using the autonity client's node key which is different comparing to the private key being used by oracle server. They are different keys.

Reply:
Ah, it uses both node key and oracke ley now `/validators/register-vali/#step-1-generate-a-cryptographic-proof-of-node-ownership`:

```
autonity genOwnershipProof --nodekey <NODE_KEY_PATH> --oraclekey <ORACLE_KEY_PATH> <TREASURY_ACCOUNT_ADDRESS>
```

I've edited this to clarify

Reply:
Can this be resolved @Jason-Zhangxin-Chen ?

Reply:
@Jason-Zhangxin-Chen

Reply:
> @Jason-Zhangxin-Chen

LGTM.